lib/matchers/associations.rb

```diff
@@ -222,7 +240,7 @@ def matches?(actual)
           end
 
           if @association[:index]
-            if metadata.index != true
+            if !metadata.indexed?
```

Comment:
`unless metadata.indexed?`

Reply:
Yeah that would be great... unless Rubocop 😅 ->  https://travis-ci.org/mongoid/mongoid-rspec/jobs/364198957

Reply:
Typical workflow for rubocop is `rubocop -a ; rubocop --auto-gen-config`.

Reply:
👍🏻